docs/api/api/api.txt

```diff
@@ -1947,6 +1947,10 @@ GET /staging/<staging_workflow_project>/staging_projects/<staging_project>
   Get the overall state of a staging project
 XmlResult: staging_project
 
+DELETE /staging/<staging_workflow_project>/staging_projects/<staging_project>
```

Comment:
why would I delete staging projects differently than regular projects? I guess I need a little context here (for the docs)

Reply:
@coolo I don't know If I understood :thinking: Do you propose to have a similar route or use exactly the same for both regular and staging projects?

Reply:
Well, I propose you clarify in the documentation why it's necessary to have an extra route and what's breaking if I use the /source route to delete the project.

Reply:
But I'm not even sure this is the case.

Reply:
actually, what prevent you to use the normal project route? :thinking:

Reply:
are you able to delete multiple projects via the ```/source``` route @coolo ?